### PR TITLE
improving stuck job checking when runtime of job run unsorted

### PR DESCRIPTION
**Pull request by chlgit**
add one unit test for the scenario that user manually start a job and it's stuck.
make test pass

### CODE REVIEW

File: bin/check_tron_jobs.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def compute_check_result_for_job_runs(client, job, job_content):
     action_run_details = client.action_runs(action_run_id.url, num_lines=10)
 
     if is_job_stuck(job_content):
-        prefix = "STUCK"
+        prefix = "Job still running when next job is scheduled to run (stuck?)"
```
Comment (solarkennedy) [inline bin/check_tron_jobs.py]: `prefix` is supposed to be the first few words. make this "WARN" and come up with a different variable to represent this sentance

**Comment by chlgit** [comment]
Here is the sample output: https://fluffy.yelpcorp.com/i/RLFg8HSdMZ2hB41QZB3Zd04H6GkC53T7.html

So far, relevant_run/action is defined as the run with state failed or succeeded.

**Comment by chlgit** [comment]
The output for stuck job looks like: https://fluffy.yelpcorp.com/i/9pt1Tk6gcgvBFggjr684pZFXQCQJ3scM.html

File: bin/check_tron_jobs.py [hunk 2]
```diff
@@ -57,35 +57,40 @@ def compute_check_result_for_job_runs(client, job, job_content):
...
 
     kwargs["output"] = (
-        "{}: {}'s last relevant run (run {}) {}.\n\n"
+        "{}: {}\n"
+        "{}'s last relevant run (run {}) {}.\n\n"
         "Here is the last action:"
         "{}\n\n"
         "And the job run view:\n"
         "{}\n\n"
         "Here are is the whole job view for context:\n"
```
[the remark this answers is not in the document]
Comment (chlgit) [reply bin/check_tron_jobs.py]: Nice catch!

**Pull request merged**